aztec/src/main/java/org/wordpress/aztec/Html.java

```diff
@@ -651,6 +655,24 @@ public void characters(char ch[], int start, int length) throws SAXException {
             }
         }
 
+        if (plugins != null) {
```

Comment:
Originally I thought I'd use it for the shortcodes (since they're essentially plaintext). However, with this approach I needed to handle the start & end tag and parse the attributes manually, which was a pain. But then I realized if I convert shortcodes to HTML elements the parser does all of this for free so I used the pre/post-processing plugins instead. I already had the text handler, though, so I thought I'd leave it here in case someone wants to create a plugin for plaintext processing, since it's already working.